core-plugins/src/e2e-test/resources/pluginParameters.properties

```diff
@@ -0,0 +1,35 @@
+projectId=cdf-athena
+dataset=test_automation
+csvFormat=csv
+
+## FILE-PLUGIN-PROPERTIES-START
+filePluginOutputFolder=file-plugin-output
+skipHeaderTrue=true
+fileSinkPathSuffix=yyyy-MM-dd-HH-mm-ss
+enableQuotedValuesTrue=true
+## FILE-PLUGIN-PROPERTIES-END
+
+## DISTINCT-PLUGIN-PROPERTIES-START
+distinctInvalidFields=$^&*
+distinctInvalidPartitions=&*^*
+distinctBQInValidFields=[{"key":"$^&*","value":""}]
+distinctBQValidFields=[{"key":"isdeleted","value":""},{"key":"longdatatype","value":""}]
```

Comment:
why do we have value = "" ? Seems like you just need an array, not a map here?

Reply:
Change done.